src/map/packets.hpp

```diff
@@ -1161,6 +1161,38 @@ struct PACKET_ZC_EL_PAR_CHANGE {
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_EL_PAR_CHANGE, 0x81e);
 
+struct PACKET_ZC_NOTIFY_ACT {  //PACKET_ZC_NOTIFY_ACT, PACKET_ZC_NOTIFY_ACT2, PACKET_ZC_NOTIFY_ACT3
+	int16 packetType;
+	int32 srcID;       // <src ID>.L
+	int32 targetID;    // <dst ID>.L
+	int32 serverTick;  // <server tick>.L
+	int32 srcSpeed;    // <src speed>.L
+	int32 dmgSpeed;    // <dst speed>.L
+#if PACKETVER < 20071113
+	uint16 damage;		// <damage>.W
+#else
+	uint32 damage;      // <damage>.L
+#endif
+#if PACKETVER >= 20131223
+	int8 isSPDamage;   // <IsSPDamage>.B
+#endif
+	uint16 div;         // <div>.W
+	uint8 type;         // <type>.B
+#if PACKETVER < 20071113
+	uint16 damage2;		// <damage2>.W
+#else
+	uint32 damage2;     // <damage2>.L
+#endif
+} __attribute__((packed));
+
+#if PACKETVER < 20071113
+DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x8a);
+#elif PACKETVER < 20131223
+DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x2e1); //ZC_NOTIFY_ACT2
+#else
+DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x8c8); //ZC_NOTIFY_ACT3
+#endif
+
```

Comment:
```suggestion
#if PACKETVER >= 20131223
struct PACKET_ZC_NOTIFY_ACT{
	int16 packetType;
	int32 srcID;
	int32 targetID;
	int32 serverTick;
	int32 srcSpeed;
	int32 dmgSpeed;
	uint32 damage;
	int8 isSPDamage;
	uint16 div;
	uint8 type;
	uint32 damage2;
} __attribute__((packed));
DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x8c8);
#elif PACKETVER >= 20071113
struct PACKET_ZC_NOTIFY_ACT{
	int16 packetType;
	int32 srcID;
	int32 targetID;
	int32 serverTick;
	int32 srcSpeed;
	int32 dmgSpeed;
	uint32 damage;
	uint16 div;
	uint8 type;
	uint32 damage2;
} __attribute__((packed));
DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x2e1);
#else
struct PACKET_ZC_NOTIFY_ACT{
	int16 packetType;
	int32 srcID;
	int32 targetID;
	int32 serverTick;
	int32 srcSpeed;
	int32 dmgSpeed;
	uint16 damage;
	uint16 div;
	uint8 type;
	uint16 damage2;
} __attribute__((packed));
DEFINE_PACKET_HEADER(ZC_NOTIFY_ACT, 0x8a);
#endif

```
Most likely damage/damage2 should not be unsigned.

Reply:
i changed to int32 again and removed they headers because it's declared on packets_struct.hpp